routes/api/v1.php

```diff
@@ -24,33 +24,35 @@
 
 //No Auth
 Route::post('login', [AuthController::class, 'login'])->name('login');
-Route::post('/students', [StudentController::class, 'store'])->name('student.create');
-Route::get('/students', [StudentController::class, 'index'])->name('students.list');
-Route::get('/students/{id}', [StudentController::class, 'show'])->name('student.show');
 
-// Students Home
+// Students for home
 Route::get('/student/list/for-home', StudentListController::class)->name('profiles.home');
 Route::get('/student/detail/for-home', StudentDetailController::class)->name('student.detail');
+// Students
+Route::get('/students', [StudentController::class, 'index'])->name('students.list');
+Route::post('/students', [StudentController::class, 'store'])->name('student.create');
+Route::get('/students/{student}', [StudentController::class, 'show'])->name('student.show');
+// TODO: Move the following two routes to Auth routes later
```

Comment:
En vez de crear to-do's, crea tarjetas en el backlog para que todos tus compañeros las puedan ver y el que quiera hacerlo que lo haga